README.rst

```diff
@@ -14,7 +14,8 @@ PAVICS
 
 Power Analytics and Visualization for Climate Science - Powered by Birdhouse and other ESGF software
 
-How to deploy the entire PAVICS platform, see the
-`README <birdhouse/README.rst>`_ and the various extra components
-`README for extra core components <birdhouse/components/README.rst>`_\ ,
-`README for optional components <birdhouse/optional-components/README.md>`_.
+For GitHub navigation, see the following README pages:
```

Comment:
Why not just keep the original text "How to deploy the entire PAVICS platform, see the following README pages:" ... cuz it reads weird if user is actually on Github.

Reply:
I mention GutHub here because on RtD, clicking these links won't work, last I checked.